tensorrt_llm/evaluate/interface.py

```
@@ -66,6 +68,47 @@ def get_model_context(llm: Any) -> tuple[str, str]:
+    if reasoning_parser.lower() == HARMONY_REASONING_PARSER:
+        # Harmony preserves final-channel boundaries in token ids, not text.
+        try:
+            from tensorrt_llm.serve.harmony_adapter import get_harmony_adapter
+
+            parsed = get_harmony_adapter().harmony_output_to_openai(
+                output.outputs[0].token_ids)
+            content = parsed.get("content")
+            if not parsed.get("_harmony_parsing_failed") and isinstance(
+                    content, str):
+                return content
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)